docs/user_guide.md

```diff
@@ -119,54 +119,65 @@ You can check one or more members, then click `ACTION`, choose one role to batch
 
 ![browse project](img/new_remove_update_member.png)
 
-## Replicating images  
-Images replication is used to replicate repositories from one Harbor instance to another.
+## Replicating resources  
+Replication allows users to replicate resources(images/charts) between Harbor and non-Harbor registries in both pull or push mode. Currently, the non-Harbor registries includes Docker Hub, Docker registry, Huawei SWR, and more registries will be supported in future.  
 
-The function is project-oriented, and once the system administrator has set a rule to one project, all repositories under the project that match the defined [filter](#image-filter) patterns will be replicated to the remote registry when the [triggering condition](#trigger-mode) is triggered. Each repository will start a job to run. If the project does not exist on the remote registry, a new project will be created automatically. If it already exists and the user configured in policy has no write privilege to it, the process will fail. The member information will not be replicated.  
+Once the system administrator has set a rule, all resources that match the defined [filter](#resource-filter) patterns will be replicated to the destination registry when the [triggering condition](#trigger-mode) is triggered. Each resources will start a task to run. If the namespace does not exist on the destination registry, a new namespace will be created automatically. If it already exists and the user configured in policy has no write privilege to it, the process will fail. The member information will not be replicated.  

```

Comment:
Each resources ->each resource;

when the triggering condition is triggered -> is matched?